src/specs/av1_isobmff/av1_isobmff.cpp

```diff
@@ -621,7 +621,7 @@ const SpecDesc specAv1ISOBMFF = {
     { "Section 2.2.4\n"
       "The config field SHALL contain an AV1CodecConfigurationBox that applies to the samples associated with this "
       "sample entry.",
-      "assert-f875c695",
+      "assert-8d3f8e0c",
```

Comment:
Looking at the code for this rule, it seems to be implementing `assert-318390e9` and `assert-a249db05`
```
Box Type:  av1C
Container: AV1 Sample Entry ('av01')
Mandatory: Yes 
Quantity:  Exactly One 
```
The proper way to implement "The config field SHALL contain an AV1CodecConfigurationBox" would be to verify that the "config" does not contain anything else than an AV1CodecConfigurationBox. The part  "that applies to the samples associated with this sample entry" is harder to implement.

Reply:
Should we put that in a separate issue to keep track?